doc/source/io.rst

```diff
@@ -2895,6 +2895,24 @@ everything in the sub-store and BELOW, so be *careful*.
 
 .. _io.hdf5-types:
 
+.. warning:: Hierarchical keys cannot be retrieved as dotted (attribute) access as described above for items stored under root node.
+
+    .. ipython:: python
```

Comment:
pls take this out. This is a completly separate issue that we may or may not include.